### PR TITLE
Html mesh

### PR DESCRIPTION
**Pull request by deltakosh**
We are promoting the HtmlMesh to our new Addons package.

### CODE REVIEW

**Comment by bjsplat** [comment]
Please make sure to label your PR with "bug", "new feature" or "breaking change" label(s).
To prevent this PR from going to the changelog marked it with the "skip changelog" label.

**Comment by bjsplat** [comment]
Please make sure to label your PR with "bug", "new feature" or "breaking change" label(s).
To prevent this PR from going to the changelog marked it with the "skip changelog" label.

**Comment by bjsplat** [comment]
Please make sure to label your PR with "bug", "new feature" or "breaking change" label(s).
To prevent this PR from going to the changelog marked it with the "skip changelog" label.

**Comment by bjsplat** [comment]
Please make sure to label your PR with "bug", "new feature" or "breaking change" label(s).
To prevent this PR from going to the changelog marked it with the "skip changelog" label.

File: packages/dev/addons/src/htmlMesh/pointerEventsCapture.ts [hunk 1]
```diff
@@ -0,0 +1,188 @@
+import { Tools } from "@babylonjs/core/Misc/tools";
```
Comment (RaananW) [inline packages/dev/addons/src/htmlMesh/pointerEventsCapture.ts:1]:
```suggestion
import { Tools } from "core/Misc/tools";
```

**Comment by bjsplat** [comment]
Please make sure to label your PR with "bug", "new feature" or "breaking change" label(s).
To prevent this PR from going to the changelog marked it with the "skip changelog" label.

**Comment by bjsplat** [comment]
Snapshot stored with reference name:
refs/pull/15783/merge

Test environment: 
https://snapshots-cvgtc2eugrd3cgfd.z01.azurefd.net/refs/pull/15783/merge/index.html

To test a playground add it to the URL, for example:

https://snapshots-cvgtc2eugrd3cgfd.z01.azurefd.net/refs/pull/15783/merge/index.html#WGZLGJ#4600

Links to test babylon tools with this snapshot:

https://playground.babylonjs.com/?snapshot=refs/pull/15783/merge
https://sandbox.babylonjs.com/?snapshot=refs/pull/15783/merge
https://gui.babylonjs.com/?snapshot=refs/pull/15783/merge
https://nme.babylonjs.com/?snapshot=refs/pull/15783/merge

To test the snapshot in the playground with a playground ID add it after the snapshot query string:

https://playground.babylonjs.com/?snapshot=refs/pull/15783/merge#BCU1XR#0

**Comment by bjsplat** [comment]
WebGL2 visualization test reporter:

https://snapshots-cvgtc2eugrd3cgfd.z01.azurefd.net/refs/pull/15783/merge/testResults/webgl2playwright/index.html

**Comment by deltakosh** [comment]
@RaananW would you mind approving?

**Comment by deltakosh** [comment]
cc @ericwood73 :)

**Review by RaananW — APPROVED** [review]

**Pull request merged**